devtools/lightning-inspect.js

```diff
@@ -221,15 +221,15 @@ window.attachInspector = function({Application, Element, ElementCore, Stage, Com
         oSetParent.apply(this, arguments);
 
         if (!window.mutatingChildren) {
-            if (parent) {
+            if (parent && parent.dhtml) {
```

Comment:
Without the check for parent.dhtml, the code can get to line 227 and throw the error below:

lightning-inspect.js:227 Uncaught (in promise) TypeError: parent.dhtml is not a function
    at StateMachineRouter.Element._setParent (lightning-inspect.js:227)
    at ElementChildList._connectParent (ElementChildList.mjs:50)